<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>276</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="settings" stretch="10,1,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="params_1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="setpoint_label">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Setpoint</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="boundry_label">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Boundary</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="gain_below_label">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Gain below boundary</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="gain_above_label">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Gain above boundary</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QDoubleSpinBox" name="setpoint">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="singleStep">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="setpoint_slider">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::NoTicks</enum>
               </property>
               <property name="tickInterval">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QDoubleSpinBox" name="boundary">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="frame">
                <bool>true</bool>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="maximum">
                <double>10000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="boundary_slider">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QSpinBox" name="gain_below">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">margin-left: 2px;</string>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="gain_below_slider">
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QSpinBox" name="gain_above">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">margin-top: 4px;</string>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::PlusMinus</enum>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="gain_above_slider">
               <property name="pageStep">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::NoTicks</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="params_2" stretch="0,1,0,1,0,1">
         <item>
          <widget class="QLabel" name="i_param_label">
           <property name="toolTip">
            <string>Expressed in .01s</string>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-top: 4px;</string>
           </property>
           <property name="text">
            <string>I-Param</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="i_param">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-top: 4px;</string>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="d_param_label">
           <property name="toolTip">
            <string>Expressed in .01s</string>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-top: 4px;
margin-left: 10px;</string>
           </property>
           <property name="text">
            <string>D-Param</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="d_param">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-top: 4px;</string>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rate_label">
           <property name="toolTip">
            <string>Rate has to be between 1 - 10</string>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-top: 4px;
margin-left: 10px;</string>
           </property>
           <property name="text">
            <string>Rate [Hz]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="rate">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-top: 4px;</string>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="toolTip">
            <string>Extern PV that can change the setpoint</string>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-right: 10px;
margin-top: 4px;</string>
           </property>
           <property name="text">
            <string>Extern Setpoint</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="extern_setpoint_on">
           <property name="styleSheet">
            <string notr="true">margin-top: 4px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="extern_setpoint">
           <property name="styleSheet">
            <string notr="true">margin-top: 4px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="formula" stretch="2,0,1,0,2,0">
         <item>
          <layout class="QVBoxLayout" name="activ_param_layout" stretch="0,0">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="activ_param_label">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Active Parameter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="activ_param">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font-size: 20px;</string>
             </property>
             <property name="text">
              <string>=</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="coefficient_layout">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="coefficient_label">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Coefficient that control the corelation between the paremeters</string>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Coefficient</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="coefficient">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">font-size: 20px;</string>
             </property>
             <property name="text">
              <string>*</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="passiv_param_layout" stretch="0,0">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="passiv_param_label">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 255);</string>
             </property>
             <property name="text">
              <string>Passive Parameter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passiv_param">
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="offset_layout"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTableWidget" name="params_table">
       <property name="minimumSize">
        <size>
         <width>415</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>415</width>
         <height>16777215</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>Parameter</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Lower Limit</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Upper Limit</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Hold Value</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
